config.env.py

```diff
@@ -1,10 +1,10 @@
-import secrets
 from os import environ as env
+from distutils.util import strtobool
 
 # Flask config
 IP = env.get('IP', '0.0.0.0')
 PORT = env.get('PORT', 8080)
-SERVER_NAME = env.get('SERVER_NAME', 'rideboard.csh.rit.edu')
```

Comment:
this is  the default when it gets deployed. Not sure this should get changed.

Reply:
This would need to be changed in case you are running local development to 127.0.0.1:PORT or whatever you're using

Reply:
Yeah, we either need to make sure that the env is set for the okd deployment, or set up a separate local config example.

Reply:
Oh I see what you mean. I meant to change that back to default rideboard before final commit. Will fix

Reply:
Changed in commit https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/711d1a4ad688e98faa8a7ae76c4c812854517b00